CAIPs/caip-275.md

```diff
@@ -199,91 +202,82 @@ However, this configuration is not recommended as having multiple wallets genera
-   If it is not supported due to platform or some other requirement, it can continue with the next one
-6. If no authentication flow can be processed by the application, then display this situation accordingly to the user.
-7. Wallet, when needed, will trigger the user for authentication and connection approval before stablishing the connection with the wallet.
-8. After wallet approves connection, application can match the domain name, the user's wallet address, and an authenticated session to the ones requested previously to validate user is not connecting with another wallet.
-9. When everything succeeds, connection should be stablished between application and the wallet found and triggered under the name provided by the user.
+4. Application filters invalid authentication flows and initiates the one that can be completed which could be by opening a new window in the user's browser to the authenticator URL, triggering the browser wallet, etc.
+   If it is not supported due to platform or some other requirement, it can continue with the next one.
+5. If no authentication flow can be processed by the application, then display this situation accordingly to the user.
+6. Wallet, when needed, will trigger the user for authentication and connection approval before establishing the connection with the wallet.
+7. After wallet approves connection, application can match the domain name, the user's wallet address, and an authenticated session to the ones requested previously to validate user is not connecting with another wallet.
+8. When everything succeeds, connection should be established between application and the wallet found and triggered under the name provided by the user.
 
 ## Connection properties
 
 How the dApp actually requests signatures and talks to the signer will vary depending on the authenticator and the chain being used.
-One option for session management is WalletConnect, where the "authenticated session" returned is actually a WalletConnect session.
+One option for session management is [Wallet Connect][], where the "authenticated session" returned is actually a WalletConnect session.
 Applications may also choose to directly integrate signer SDKs, providing a more streamlined signing flow.
 
 For example, here is how an application would integrate with the Login With Name login process:
 
-1. Install wagmi and [Login With Name WAGMI connector][] packages
-2. Import the loginWithName connector from the package
-3. Configure the wagmi config with the loginWithName connector.
-   This package provides ENS as a name resolver which can be configured to use mainnet or sepolia
-4. Wrap the app with WagmiProvider, pass it the configuration with the loginWithName connector
-5. Upon connection request, the connector will request application for user name. Application can show user an input for it if not obtained in other way.
+1. Install [Wagmi][] and [Login With Name WAGMI connector][] packages.
```

Comment:
```suggestion
1. Install connection/authentication libraries, such as the [Wagmi][] and [Login With Name WAGMI connector][] packages.
```
Even tho it's a non-normative example, still good to generalize so that the spec still makes sense years from now, if, say, wagmi gets renamed, splits in two, etc etc.